.claude/skills/scribe/SKILL.md

```
@@ -0,0 +1,61 @@
+
+   Read it and extract frontmatter.
+
+4. **Update task status** — edit the task file frontmatter:
+
+   Change `status: todo` → `status: done`
+
+5. **Find the merged PR** for this task:
+
+   ```
+   gh pr list --state merged --search "T$ARGUMENTS" --limit 1 --json number,title,url
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)